incidents/test_transform.py

```diff
@@ -0,0 +1,57 @@
+"""Unit tests to test transform functions"""
+import pandas as pd
+
+from transform import process_xml, replace_new_line, convert_timestamps
+from test_data.test_vars import (OUTPUT_AFTER_PROCESSING_XML,
```

Comment:
Excellant use of importing test varibales into your test file, makes the file easier to read and definately less clumpy